index.html

```diff
@@ -173,6 +173,21 @@ <h2>About <br> Myself</h2>
                     <li class="list-item"><a href="#">Skills</a> <i class="fa fa-angle-right"></i></li>
                 </ul>
             </section>
+            <section class="contact_form">
+              <div class="contact-div">
+                <h2 class="contact-h2">Contact me</h2>
+                <p class="contact-paragraph">
+                    If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it 
+                </p>
+                <form action="https://formspree.io/f/form_id" method="post">
```

Comment:
- [ ] Kindly your form still not working, First you have to create an account on [Formspree service](https://formspree.io/html) after that add a new form like in this pic
![image](https://user-images.githubusercontent.com/35535130/159598607-d4a0045f-e082-46ef-a6e3-b80bc8599aaf.png)

After that open, the integration tab and you will find your link with the id. Take it cope and replace it with this line `https://formspree.io/f/form_id`.

![image](https://user-images.githubusercontent.com/35535130/159598789-8bbac3c8-2a8a-4909-80aa-e5af88aa9cd2.png)

Try to test your form first before you request a review again the page after submitting should say `Thank you` work, not `Form not found.

Your page currently says after submitting 
![image](https://user-images.githubusercontent.com/35535130/159600097-08d91f5d-9d52-44fa-9998-0b53007f013d.png)